centreon/src/App/Upgrade/Infrastructure/Dbal/DbalModuleRepository.php

```
@@ -292,9 +292,8 @@ private function executeModulePhpFile(string $filePath): void
             return;
         }
 
-        // Variables expected by legacy module upgrade scripts.
-        $pearDB = $this->configConnection->getNativeConnection();
-        $pearDBStorage = $this->realtimeConnection->getNativeConnection();
+        $pearDB = $this->configConnection;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$pearDB'. (undefined)

(UnusedLocalVariable)